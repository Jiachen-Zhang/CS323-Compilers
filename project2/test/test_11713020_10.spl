struct Apple {
  int size;
};
int fun(struct Apple apple) {
  return 1;
}

int test_11713020_8()
{
  struct Apple apple;
  fun(apple);
  fun1(apple);
  return apple;
}