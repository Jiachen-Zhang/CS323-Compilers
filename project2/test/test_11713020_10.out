Error type 3 at Line 10: redefine variable: apple
Error type 2 at Line 12: undefined function: fun1
Error type 8 at Line 13: incompatiable return type
